index.md

```diff
@@ -79,6 +79,17 @@ The easiest way to get started with Zowe is by downloading the convenience build
   {% endif %}
 {% endfor %}
 </details>
+<details>
+<summary>Nightly Builds</summary>
+<p>Please visit Zowe Artifactory nightly build folder to find the most recent build: <a class="white" href="https://zowe.jfrog.io/zowe/webapp/#/artifacts/browse/tree/General/libs-release-local/org/zowe/nightly">https://zowe.jfrog.io/zowe/webapp/#/artifacts/browse/tree/General/libs-release-local/org/zowe/nightly</a>.</p>
+<p>Please note:
+  <ul>
+    <li><i>Nightly builds might not be stable, it's for preview purpose.</i></li>
```

Comment:
Rephrase to `Nightly builds are made available to allow preview and early distribution of in-progress work items which may be functionally incomplete and unstable.  The coverage and successful execution of tests has not been guaranteed and the builds should be treated accordingly.`